controllers/discordactions.js

```diff
@@ -168,6 +165,7 @@ const deleteRole = async (req, res) => {
     if (!roleExists || req.userData.id !== userData.user.id) {
       res.boom.forbidden("Permission denied. Cannot delete the role.");
     }
+    await discordServices.removeRoleFromUser(roleid, userid, req.userData);
```

Comment:
why this one is added?

Reply:
> This function call was earlier in the removeMemberGroup function. Changing it to decouple the db operation and discord API call

are we covering and testing well..?

Reply:
Yes, discordServices.removeRoleFromUser is already test covered.